### DISCUSSION TITLE
How to customize the Bot_Hello text for different masks

### DISCUSSION THREAD

**Discussion by Sushang-Li** [ChatGPTNextWeb/NextChat#3075] [Q&A]
I want to customize the Bot_Hello text for different masks to alert the user, but I don't want to show all the in-context, so I want to customize the bot_hello text for different masks. And I have tried many ways, but it doesn't work. Can someone help me? Thanks

**Comment by GOvEy1nw** [comment]
> I want to customize the Bot_Hello text for different masks to alert the user, but I don't want to show all the in-context, so I want to customize the bot_hello text for different masks. And I have tried many ways, but it doesn't work. Can someone help me? Thanks

I don't have an answer, but I agree this is a feature that should be implemented. That way the user can be given some basic info regarding what the mask is and what its purpose is etc, without having to read all of the context prompts.

I think a possible solution, that hopefully shouldn't be too much trouble to implement, would be to allow us to hide some context prompts and show others. That way we could hide all except a final 'assistant' context prompt that says whatever we want it to say, if that makes sense?
